project1/styles.css

```
@@ -0,0 +1,123 @@
+body {
+    background-color: hsl(0, 0%, 96%);
+}
+
+header {
+    width: 80%;
+    margin: auto;
+}
+
+main {
+    margin-left: auto;
+    margin-right: auto;
+    width: 60%;
+    line-height: 1.8;
+}
+
+#octopicture {
+    width: 70%;
+    margin: auto;
+}
+
+figure {
+    margin: auto;
+    width: 60%;
+}
+
+img {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 100%;
+}
+
+h1 {
+    font-family: 'Roboto Slab', serif;
+    font-size: 62px;
+    font-weight: bolder;
+    color: hsl(177, 32%, 51%);
+    text-align: left;
+}
+
+h2 {
+    font-family: 'Roboto Slab', serif;
+    font-size: 32px;
+    font-weight: bold;
+    color: hsl(0, 0%, 20%);
+}
+
+h3 {
+    font-family: 'Roboto Slab', serif;
+    font-size: 18px;
+    font-weight: normal;
+    color: hsl(0, 0%, 20%);
+}
+
+h4 {
+    font-family: 'Roboto Slab', serif;
+    font-size: 18px;
+    font-weight: normal;
+    color: hsl(0, 0%, 20%);
+}
+
+p {
+    font-family: 'Roboto', regular;
+    font-size: 18px;
+    font-style: normal;
+    color: hsl(0, 0%, 20%);
+    text-align: left;
+    width: auto;
+
+}
+
+q {
+    font-family: 'Roboto', regular;
+    font-size: 22px;
+    font-style: normal;
+    color: rgb(255, 255, 255);
+    text-align: center;
+    width: auto;
+}
+
+/*#video {}*/
+
+/*#wrapper {}*/
+
+iframe {
+    display: block;
+    /*float: left;*/
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 20px;
+
+}
+
+#pullquote {
+    margin-top: 10px;
+    margin: auto;
+    /*float: right;*/
+    padding: 3%;
+    width: 40%;
+    background-color: hsl(177, 32%, 51%);
+}
+
+figcaption {
+    font-family: 'Roboto', regular;
+    font-size: 15px;
+    font-style: italic;
+    /*how do i specify that I want this to be italic AND thin*/
+    color: hsl(0, 0%, 20%);
+    text-align: center;
+    width: auto;
+}
+
+ul {
+    font-family: 'Roboto', regular;
```

```suggestion
    font-family: 'Roboto', sans-serif;
```

Instead of `regular` what you want here is to set a fallback if, for example, the Google webfonts server was down. With the code above, your page would display a system default sans-serif font.

Note that you'll want to make this change everywhere you reference Roboto.